scripts/base/frameworks/notice/main.zeek

```diff
@@ -585,7 +585,7 @@ function is_being_suppressed(n: Notice::Info): bool
 
 # Executes a script with all of the notice fields put into the
 # new process' environment as "ZEEK_ARG_<field>" variables.
-function execute_with_notice(cmd: string, n: Notice::Info)
+function execute_with_notice(cmd: string, n: Notice::Info) &deprecated
```

Comment:
Should there be warning text for all of these deprecated script functions? Something like "Remove in v6.1. Usage testing indicates this function is unused" would be good enough.

Reply:
Yeah, that's a good thought.  Want to go ahead and add that, or should I?

Reply:
Have at it. I'm tracking down some other stuff today.

Reply:
Just pushed